Jenkinsfile

```diff
@@ -25,19 +18,25 @@ pipeline {
                 sh "npm install"
             }
         }
-
         stage('Create Files') {
             steps {
-                sh "curl http://localhost:8088/repository/content-media/ml-media/files.zip --output ${WORKSPACE}/files.zip"
-                sh "unzip files.zip"
-                sh "mv files __test__"
+                sh "curl http://localhost:8088/repository/content-media/ml-media/files.zip --output ${WORKSPACE}/__test__/files.zip"
+                //sh "cd __test__/; yes | unzip files.zip"
```

Comment:
You should remove all the comment lines

Reply:
Oh, thanks David. Thanks for noticing me. now I correct it

Reply:
Nice Florencia!